draft/spec/index.html

```diff
@@ -868,6 +868,28 @@ <h2>Extensions</h2>
             </p>
         </blockquote>
     </section>
+    
+    <section id="filesystem-features">
+        <h2>Filesystem features</h2>
+        <p>
+            In order to maximize the compatibility of the OCFL with different filesystems, and thus improve the
+            portability of OCFL Objects between different systems, some restrictions on the use of certain
+            filesystem features are necessary. If the preservation of non-OCFL-compliant features is required then
+            data MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular
+            file.
+        </p>
+        <ol>
+          <li>Filesystem metadata (e.g. permissions, access, and creation times) are not considered portable between
+              filesystems or preservable through file transfer operations. Nor can these attributes be validated in
```

Comment:
'Nor can' => 'These attributes also cannot be validated ... '

Reply:
:+1: